libbpf-tools/README.md

```diff
@@ -92,3 +92,30 @@ CONFIG_DEBUG_INFO=y
 kernel build (it comes from dwarves package). Without it, BTF won't be
 generated, and on older kernels you'd get only warning, but still would
 build kernel successfully
+
+Running in kernels without CONFIG_DEBUG_INFO_BTF=y
+--------------------------------------------------
+
+It's possible to run some tools in kernels that don't expose
+`/sys/kernel/btf/vmlinux`. For those cases,
+[BTFGen](https://lore.kernel.org/bpf/20220215225856.671072-1-mauricio@kinvolk.io)
+and [BTFHub](https://github.com/aquasecurity/btfhub) can be used to
+generate small BTF files for the most popular Linux distributions that
+are shipped with the tools in order to provide the needed information to
+perform the CO-RE relocations when loading the eBPF programs.
+
+If you haven't cloned the
+[btfhub-archive](https://github.com/aquasecurity/btfhub) repository, you
+can run make and it'll clone it for you into the `$HOME/.local/share`
+directory:
+
+```bash
+ENABLE_MIN_CORE_BTFS=1 make -j$(nproc)
```

Comment:
similar here, it seems customary to do it like this:

```
make ENABLE_MIN_CORE_BTFS=1 -j$(nproc)
```

There are also subtle semantic differences, as this way ENABLE_MIN_CORE_BTFS will override any `ENABLE_MIN_CORE_BTFS ?= <my default value>` definitions, while the `ENABLE_MIN_CORE_BTFS=1 make` won't (at least that's why I remember from times when I had to deal with Makefiles a bit more).